src/navier_stokes/SpongeLayerForceFunction.cpp

```diff
@@ -98,6 +98,18 @@ SpongeLayerForceFunction::SpongeLayerForceFunction(const std::string& object_nam
                 d_width[location_index] = input_db->getDouble(width_key);
             }
         }
+        if (input_db->keyExists("chi"))
+        {
+            d_chi = input_db->getDouble("chi");
+        }
+        else
+        {
+            d_chi = 1.0;
+            TBOX_WARNING(
+                "SpongeLayerForceFunction::SpongeLayerForceFunction():"
+                << "Penalization factor for sponge layer not specified in the input file. Using a default value of 1.0"
+                << std::endl);
```

Comment:
Why can't we set things up to use the same default value as before? This is going to change people's codes since its no longer `0.5 * rho`.

Reply:
But this value `0.5 * rho` can be set through the input file. It is a penalty term whose function is to dampen out vortices at the outflow boundary. The precise value does not matter. 

Are there an existing code in IBAMR that actually uses this class?

Reply:
I am not aware of any code that uses it but if its not too hard I would prefer not to change their results.

Reply:
Noticed this in my inbox and thought I'd chime in:

You could condition check to see if the constant coefficient fluid solver is being used. We do something similar in ConstraintIBMethod:
https://github.com/IBAMR/IBAMR/blob/master/src/IB/ConstraintIBMethod.cpp#L527

If so, then you can use getRho() and set kappa to 0.5*rho. If not, then require it to be specified in the input file. This would ensure that existing applications don't break.

Reply:
We should not change the default behavior, and we should try to avoid having different default behavior for different classes.

If the default behavior needs to be different when used with different classes, then I would suggest to have two different implementations --- a constant coefficient implementation and a variable coefficient implementation.